src/NServiceBus.RabbitMQ.Tests/APIApprovals.cs

```diff
@@ -13,7 +14,9 @@ public class APIApprovals
     [MethodImpl(MethodImplOptions.NoInlining)]
     public void Approve()
     {
-        var publicApi = Filter(ApiGenerator.GeneratePublicApi(typeof(RabbitMQTransport).Assembly));
+        var combine = Path.Combine(TestContext.CurrentContext.TestDirectory, "NServiceBus.Transports.RabbitMQ.dll");
+        var assembly = Assembly.LoadFile(combine);
+        var publicApi = Filter(ApiGenerator.GeneratePublicApi(assembly));
```

Comment:
because given your test runner typeof(RabbitMQTransport).Assembly might resolve to somewhere where dependency dlls do not exist. it was failing for me